### ISSUE TITLE
Design issue on 4K Screens

### ISSUE THREAD

**Issue by ashverma13** [flarum/core#1203]
Hey,

There is an issue on the responsive design for 4K screens.

I'm attaching a screenshot from your demo, to make sure it wasn't our css messing it up.

Thanks,
![flarum](https://user-images.githubusercontent.com/14328642/27706865-d4fbfc84-5d30-11e7-9e5f-c20b8614e40c.png)

**Comment by tobscure** [comment]
What browser?

**Comment by luceos** [comment]
It seems to scale to the mobile version @ashverma13 what resolution is that 4k screen?

**Comment by ashverma13** [comment]
The full screen resolution is 3240px wide. It shows the error only on 4K display. On standard monitors it does scale to mobile view. But on 4K display it shows this error and only for 768px width

**Comment by ashverma13** [comment]
This issue happens on all the browsers

**Comment by luceos** [comment]
@ashverma13 is this also happening on the http://discuss.flarum.org ? And are you able to open the browser debug console to see any javascript errors are popping up?

**Comment by ashverma13** [comment]
@luceos Yes, this is also happening on the http://discuss.flarum.org. Please refer to the screenshot in the original post. Also, there are no javascript errors in debug console.

**Comment by adamtester** [comment]
Just tried on my 4k screen, seems to be fine, I don't think its a resolution issue

**Comment by CDK2020** [comment]
I can confirm this happens, it happens for me when I snap the window to half the screen. The resolution would be 1920 x 2160 (width x length)

**Comment by datitisev** [comment]
I can't confirm this with my iMac. I've tried with full window size, full screen, and half screen.

**Comment by datitisev** [comment]
Can anyone confirm that this is still an issue, or is it fixed?

**Comment by CDK2020** [comment]
Doesn't seem to be happening anymore
